duckduckgo/iOS
# Default GPC to on

Task/Issue URL: https://app.asana.com/0/1197921851987065/1199132975397327/f
Tech Design URL:
CC: @brindy @catehstn 

**Description**:
This PR defaults the GPC setting to on.

**Steps to test this PR**:
1. Fresh install app
1. Navigate to Settings > GPC
1. Verify GPC is on



**Device Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
